test/definitions/page.js

```diff
@@ -12,13 +12,14 @@ var definition = function() {
   };
   this.search = {
     mapping: {
-      'location.geo': { type: 'geo_point', 'lat_lon': true }
+      'location.geo': { type: 'geo_point' },
+      url: {type: 'keyword'}
     }
   };
   this.schema = {
     title: {type: String, required: true},
     body: {type: String},
-    url: {type: String, elastic: {mapping: {type: 'string', index: 'not_analyzed'}}},
+    url: {type: String},
```

Comment:
hui. Das sieht aber gefährlich aus.... früher war das verdammt wichtig dass bei einer url not analyzed steht....

Reply:
Sollte passen mit elastic 5.X changes (https://travis-ci.org/organizations/molecuel/repositories), unless I misunderstood something there;

Reply:
Was passiert denn wenn der den Index aufbaut? früher hat der den string komplett analysiert und megaviel daten erzeugt im Index. Passiert das sicher nicht?

Reply:
der `keyword` Datentyp wird laut [elasticsearch Doku](https://www.elastic.co/guide/en/elasticsearch/reference/6.2/keyword.html) nie analysiert: "Keyword fields are only searchable by their exact value." 
Im vergleich dazu wird der Datentyp `text` durch einen Analyzer gejagt, siehe [Doku dazu](https://www.elastic.co/guide/en/elasticsearch/reference/6.2/text.html).

Reply:
Dann ist doch fein!

Reply:
(Der mapping type ist in den search-Abschnitt des Page-Models umgezogen, wie es auch aktuell im csi2015 prod der fall ist)